drivers/pinmux/pinmux_esp32.c

```diff
@@ -123,16 +123,19 @@ static int pinmux_input(struct device *dev, u32_t pin, u8_t func)
 	volatile u32_t *reg;
 	int r;
 
+	/* Since PINMUX_INPUT_ENABLED == 1 and PINMUX_OUTPUT_ENABLED == 0,
```

Comment:
What would happen if one wanted a pin to be used as output?

Reply:
I tested the gpio sample and it worked, but I'm not sure if there will be problems under other conditions. I'm going to try another approach to using pins as IO.

Reply:
Can I add the flags below so we can set the pin as IO and avoid the workaround? 

`#define GPIO_DIR_IN_OUT (1 << 18)` on _gpio.h_

`#define  PINMUX_INPUT_OUTPUT_ENABLED (0x2)` on _pinmux.h_

Reply:
I believe that API changes at this level requires some more input. We're reviewing a lot of our APIs, so it's good to have feedback like this. Please open an issue and link it to #5697.